grades/templates/course_list.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'layouts/default/page.html' %} {% load bootstrap4 %} 
+{% load render_table from django_tables2 %} {% block content %}
+<h2>Your Courses</h2>
+
+<h5>Add a new course here:</h5>
```

Comment:
I'm not 100% following. 
shouldn't the user choose a course from a list of available courses and assign it to him with the details relevant to him (grade, etc)